src/GafferUI/AnimationGadget.cpp

```diff
@@ -633,41 +633,43 @@ void AnimationGadget::renderLayer( Layer layer, const Style *style, RenderReason
 
 				// draw the tangents
 				//
-				// NOTE : only draw if they are unconstrained and key or adjacent key is selected
+				// NOTE : only draw if they are unconstrained and key or adjacent key is selected or opposite tangent is being moved
```

Comment:
Might help to point out that the mechanism for this is that isOutHighlighted gets set true when the opposite tangent is being moved - that's fairly obvious when looking at this commit, but might not be when looking at this code in the future.

Reply:
expanded the comments.